flink-catalog-aws-glue/src/main/java/org/apache/flink/table/catalog/glue/GlueCatalogConstants.java

```diff
@@ -0,0 +1,249 @@
+     * HTTP_CLIENT_TYPE_DEFAULT will be used. For specific types supported, see HTTP_CLIENT_TYPE_*
+     * defined below.
+     */
+    public static final String HTTP_CLIENT_TYPE = "http-client.type";
+
+    /**
+     * Used to configure the connection acquisition timeout in milliseconds for {@link
+     * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when
+     * {@link #HTTP_CLIENT_TYPE} is set to HTTP_CLIENT_TYPE_APACHE
+     *
+     * <p>For more details, see
+     * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+     */
+    public static final String HTTP_CLIENT_APACHE_CONNECTION_ACQUISITION_TIMEOUT_MS =
+            "http-client.apache.connection-acquisition-timeout-ms";
+
+    /**
+     * If Glue should skip name validations It is recommended to stick to Glue best practice in
+     * https://docs.aws.amazon.com/athena/latest/ug/glue-best-practices.html to make sure operations
+     * are Hive compatible. This is only added for users that have existing conventions using
+     * non-standard characters. When database name and table name validation are skipped, there is
+     * no guarantee that downstream systems would all support the names.
+     */
+    public static final String GLUE_CATALOG_SKIP_NAME_VALIDATION = "glue.skip-name-validation";
+
+    /**
+     * Used to configure the connection max idle time in milliseconds for {@link
+     * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when
+     * {@link #HTTP_CLIENT_TYPE} is set to HTTP_CLIENT_TYPE_APACHE
+     *
+     * <p>For more details, see
+     * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+     */
+    public static final String HTTP_CLIENT_APACHE_CONNECTION_MAX_IDLE_TIME_MS =
+            "http-client.apache.connection-max-idle-time-ms";
+
+    /**
+     * Used to configure the connection time to live in milliseconds for {@link
+     * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when
+     * {@link #HTTP_CLIENT_TYPE} is set to HTTP_CLIENT_TYPE_APACHE
+     *
+     * <p>For more details, see
+     * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+     */
+    public static final String HTTP_CLIENT_APACHE_CONNECTION_TIME_TO_LIVE_MS =
+            "http-client.apache.connection-time-to-live-ms";
+
+    /**
+     * Used to configure the connection timeout in milliseconds for {@link
+     * software.amazon.awssdk.http.apache.ApacheHttpClient.Builder}. This flag only works when
+     * {@link #HTTP_CLIENT_TYPE} is set to HTTP_CLIENT_TYPE_APACHE
+     *
+     * <p>For more details, see
+     * https://sdk.amazonaws.com/java/api/latest/software/amazon/awssdk/http/apache/ApacheHttpClient.Builder.html
+     */
+    public static final String HTTP_CLIENT_APACHE_CONNECTION_TIMEOUT_MS =
```

Comment:
All of the AWS SDK/client specific things should ideally go in the AWS base module so they can be reused across projects. We are aiming for consistent configuration keys across AWS integrations. Can you please refactor?